proto/babylon/finality/v1/tx.proto

```diff
@@ -25,6 +25,9 @@ service Msg {
     rpc UnjailFinalityProvider(MsgUnjailFinalityProvider) returns (MsgUnjailFinalityProviderResponse);
```

Comment:
remove `    // TODO: msg for evidence of equivocation. this is not specified yet`